app/components/post_textbox/post_textbox_base.js

```diff
@@ -106,6 +106,7 @@ export default class PostTextBoxBase extends PureComponent {
             top: 0,
             value: props.value,
             channelTimezoneCount: 0,
+            isMessageTooLong: false,
```

Comment:
nit: I'd be more explicit and say, `longMessageAlertShown` or some such.

Reply:
Done!